src/contrib/serializers/Akka.Serialization.Hyperion/reference.conf

```diff
@@ -0,0 +1,27 @@
+﻿akka.actor {
+	serializers {
+		hyperion = "Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion"
+	}
+	serialization-settings {
+		hyperion {
+			# If true, hyperion serializer will preserve references between objects. 
+			# This is necessary to support things such as cyclic dependencies between objects.
+			preserve-object-references = true
```

Comment:
Need specs to validate the defaults (protects against regressions where the defaults get changed by accident)